c_reference/include/quantized_utils.h

```diff
@@ -20,7 +20,21 @@ inline INT_T saturate(INTM_T inp) {
     }
 }
 
+// This function is used to provide a truncation of input to a specific
+// range within the ReLU operation.
+inline INTM_T q_relu(INTM_T inp, INTM_T limit) {
+    if (inp > limit){
+        return limit;
+    } else if (inp < 0) {
+        return 0;
+    } else {
+        return inp;
+    }
+}
+
```

Comment:
check with Arun's PR whether you are redefining relu

Reply:
Arun's PR doesn't take a limit variable for `ReLU`, plus it takes an entire `INT_T` matrix as an input. I think the better thing to do here is to merge this as is, and make further modifications when we're rebasing Arun's commits to master.